### DISCUSSION TITLE
GPU usage for default tensforflow backend (on Google Colab)

### DISCUSSION THREAD

**Discussion by Aries744** [lululxvi/deepxde#1190] [Q&A]
Has anyone seemed to not see an increase in GPU RAM being used when training a DeepXDE model on Google Colab?

--------
import tensorflow as tf
import os
dde.backend.backend.is_gpu_available()
if tf.test.gpu_device_name():
    print("GPU found: ", tf.test.gpu_device_name())
else:
    print("No GPU found")

output:
GPU found:  /device:GPU:0
--------

This shows me that tensorflow should be using this GPU for all its operations in the DeepXDE backend. Has anyone else had this problem?

**Comment by lululxvi** [comment]
Maybe try a larger training data to check?
